h/browser/chrome/karma.config.js

```diff
@@ -18,6 +18,15 @@ module.exports = function(config) {
       // Polyfills for PhantomJS
       '../../../node_modules/js-polyfills/polyfill.js',
 
+      {
+        pattern: 'test/settings.json',
+        watched: true,
+        included: false,
+        served: true,
+        nocache: false
```

Comment:
The values of `watched`, `served`, and `nocache` are the defaults and can be omitted.